sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing/Program.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            GenerateTraces();
+            GenerateLogs();
+            GenerateMetrics();
+        }
+
+        private static void GenerateTraces()
+        {
+            ActivitySource source = new ActivitySource("MyCompany.MyProduct.MyLibrary");
+
+            var resourceAttributes = new Dictionary<string, object> { { "service.name", "my-service" }, { "service.namespace", "my-namespace" }, { "service.instance.id", "my-instance" } };
+            var resourceBuilder = ResourceBuilder.CreateDefault().AddAttributes(resourceAttributes);
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                            .SetResourceBuilder(resourceBuilder)
+                            .AddSource("MyCompany.MyProduct.MyLibrary")
+                            .AddAzureMonitorTraceExporter(o =>
+                            {
+                                o.ConnectionString = $"InstrumentationKey=Ikey;";
```

Comment:
```suggestion
                                o.ConnectionString = $"InstrumentationKey=00000000-0000-0000-0000-000000000000";
```

nit: i recommend using this for the example. This matches what is in our docs.